src/analyses/apron/affineEqualityAnalysis.apron.ml

```diff
@@ -34,4 +34,4 @@ let after_config () =
   GobConfig.set_string "ana.path_sens[+]"  (Spec.name ())
 
 let _ =
-  AfterConfig.register after_config
+  AfterConfig.register after_config
```

Comment:
This contains white-space changes only, and makes the diff unnecessarily large. Please revert.